src/main/resources/portal.properties.EXAMPLE

```diff
@@ -220,20 +220,35 @@ ucsc.build=hg19
 oncoprint.defaultview=patient
 
 # OncoPrint driver mutation annotations
-#   oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
-#   oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
-#   oncoprint.custom_driver_annotation.binary.default=true
-#   oncoprint.custom_driver_annotation.tiers.default=true
-  oncoprint.oncokb.default=true
-  oncoprint.hotspots.default=true
-#  oncoprint.hide_vus.default=true
+# oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
+# oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
+# oncoprint.custom_driver_annotation.binary.default=true
+# oncoprint.custom_driver_annotation.tiers.default=true
+oncoprint.oncokb.default=true
+oncoprint.hotspots.default=true
+# oncoprint.hide_vus.default=true
 
 # Custom gene sets
 # querypage.setsofgenes.location=file:/<path>
-#
-# Disable cache (for non authorized portals this can sometimes avoid running
-# out of memory with relatively little performance loss)
-# mybatis.cache.enabled=false
+
+# to disable use of Ehcache, set ehcache.cache.enabled to false
+ehcache.cache_enabled=true
+ehcache.xml_configuration=/ehcache.xml
+ehcache.statistics_enabled=true
+# Enable cache statistics endpoint for Ehcache monitoring
+cache.statistics_endpoint_enabled=false
+
+# Properties for specifying cache size constraints
+# - Zero is not an accepted value, will break deployment
+# - disk size (if used) must be greater than heap size
+
+ehcache.general_repository_cache.max_mega_bytes_heap=5120
+ehcache.static_repository_cache_one.max_mega_bytes_heap=30
```

Comment:
these defaults should prolly also be set in code. Ideally people should be able to run the new code without having to update their properties file. A good CI test would be to pull the existing EXAMPLE properties file from the branch one is running against, in this case `master`, compile & run with that properties file and see if one is able to boot the portal.

Reply:
done